riscv/src/continuations/bootloader.rs

```diff
@@ -630,8 +630,10 @@ pub fn default_input<T: FieldElement>(accessed_pages: &[u64]) -> Vec<Elem<T>> {
     // Set all registers and the number of pages to zero
     let register_values = default_register_values();
 
+    let merkle_tree = MerkleTree::<T>::new();
+
     if accessed_pages.is_empty() {
-        let empty_hash = MerkleTree::<T>::empty_hash();
+        let empty_hash = *merkle_tree.root_hash();
```

Comment:
This whole function can now be:
```rust
pub fn default_input<T: FieldElement>(accessed_pages: &[u64]) -> Vec<Elem<T>> {
    // Set all registers and the number of pages to zero
    let register_values = default_register_values();
    let merkle_tree = MerkleTree::<T>::new();

    // TODO: We don't have a way to know the memory state *after* the execution.
    // For now, we'll just claim that the memory doesn't change.
    // This is fine for now, because the bootloader does not yet enforce that the memory
    // state is actually as claimed. In the future, the `accessed_pages` argument won't be
    // supported anymore (it's anyway only used by the benchmark).
    create_input(
        register_values,
        &merkle_tree,
        accessed_pages.iter().map(|&x| x as u32),
    )
}
```
Should be exactly the same.

Reply:
Done.